pages/_type/_id/version.vue

```diff
@@ -698,6 +728,20 @@ export default {
     }
   },
   methods: {
+    checkFields() {
+      if (
+        this.version.version_number === '' ||
+        this.version.game_versions.length === 0 ||
+        this.version.loaders.length === 0
+      ) {
+        return false
+      }
+      if (this.newFiles.length === 0 && this.version.files.length === 0) {
+        return false
+      }
```

Comment:
Can be combined with the above statement via parentheses, doesn't really matter though

Reply:
fixed